aliases.php

```diff
@@ -1,31 +1,30 @@
 <?php
+
+use Winter\Storm\Support\ClassLoader;
+
 /**
  * To allow compatibility with plugins that extend the original RainLab.User plugin, this will alias those classes to
  * use the new Winter.User classes.
  */
 $aliases = [
     // Reverse alias to fix issue on PHP 7.2, see https://github.com/wintercms/wn-user-plugin/runs/2122181184
-    'RainLab\Notify\Classes\EventBase'          => Winter\Notify\Classes\EventBase::class,
+    RainLab\Notify\Classes\EventBase::class     => Winter\Notify\Classes\EventBase::class,
```

Comment:
Is the reverse alias still required with the new lazy class aliasing? @mjauvin @bennothommo

Reply:
I would think so @LukeTowers. This shouldn't change anything in this regard, unless I'm mistaken @bennothommo ?

Reply:
@bennothommo care to confirm or refute this?

Reply:
@mjauvin @LukeTowers we *won't* need the reverse alias anymore. The only reason we had it was because the `UserEventBase` class was being instantiated by the `class_alias` call originally, which was requiring `EventBase`. Now that it's lazy loaded, these classes won't be hit at all unless the Notify plugin is installed.